src/asymmetric_crypto_ctx.cpp

```diff
@@ -400,7 +400,11 @@ namespace mococrw {
      * #  ECDSA  #
      * ###########
      */
-
+namespace {
+    ECDSASignatureFormat SDAtoDSASignatureFormat (ECSDASignatureFormat sigFormat) {
```

Comment:
```suggestion
    /**
     * This is required for backwards compatibility, since we had a typo in ECDSA for a while.
     */
    ECDSASignatureFormat SDAtoDSASignatureFormat(ECSDASignatureFormat sigFormat) {
```